index.html

```diff
@@ -969,24 +983,24 @@ <h6 class="font-weight-bolder mt-4">In-depth Stories You Don't See Elsewhere Abo
               </section>
             </div>  
   
-            <div class="col-lg-5 mt-3">   
-               <h6 class="after-cover-text subs-title news-6 pt-2 ml-3">NEWSLETTER</h6>     
-              <section class="container-fluid mb-3">
+            <div class="col-lg-4 mt-3">   
+               <h6 class="after-cover-text subs-title news-6 pt-2 ml-1">NEWSLETTER</h6>     
+              <section class="container-fluid mb-5">
```

Comment:
Please add padding around these and align them correctly as present on the original page.